homeassistant/components/plugwise/climate.py

```diff
@@ -121,21 +155,43 @@ def extra_state_attributes(self) -> Mapping[str, Any] | None:
             "selected_schema": self.device.get("selected_schedule"),
         }
 
+    @property
+    def supported_features(self) -> int:
+        """Return the supported features."""
+        features: int = ClimateEntityFeature.TARGET_TEMPERATURE
+        if self.hc_data.get("elga_cooling_enabled", False):
+            features = ClimateEntityFeature.TARGET_TEMPERATURE_RANGE
+        if self.device.get("preset_modes"):
+            features |= ClimateEntityFeature.PRESET_MODE
+
+        return features
```

Comment:
The problem is, the owner of an Elga can decide to turn the cooling-feature on or off at any time he/she wants. This will not happen often but it is possible.
If this happens and HA Core is not restarted/reloaded directly after such a switch, the integration will not function correctly.

Reply:
> The problem is, the owner of an Elga can decide to turn the cooling-feature on or off at any time he/she wants. This will not happen often but it is possible.

Shouldn't they still show up, but cause an error when selecting?

Reply:
Adapted to review feedback.